src/ensembl/src/content/home/Home.tsx

```diff
@@ -22,12 +22,12 @@ const Home: FunctionComponent<HomeProps> = (props: HomeProps) => {
         <h2>Previously viewed</h2>
         <dl>
           <dd>
-            <Link to="/app/browser/human/BRCA2/13:32315474-32400266">
+            <Link to="/app/browser/human/BRCA2/?region=13:32315474-32400266">
```

Comment:
_(here and below)_ Perhaps remove the slash after the gene name?

Reply:
That's done.